client/homebrew/pages/userPage/userPage.jsx

```diff
@@ -14,69 +14,50 @@ const HelpNavItem = require('../../navbar/help.navitem.jsx');
 const ErrorNavItem = require('../../navbar/error-navitem.jsx');
 const VaultNavitem = require('../../navbar/vault.navitem.jsx');
 
-const UserPage = createClass({
-	displayName     : 'UserPage',
```

Comment:
Just leaving a note that possibly we may want to add something like:

```
UserPage.displayName = 'UserPage'
```
at the end of the component (after the closing bracket).  My understanding right now is that the build process may automatically be doing this, inferring the name from the function name, so we probably *don't* need to do this.  So really, just a note that if we start to notice in error messages that the component name isn't showing, we can come back and add this ourselves.

Reply:
Hmm, good point, from the docs:

> The displayName string is used in debugging messages. Usually, you don’t need to set it explicitly because it’s inferred from the name of the function or class that defines the component. You might want to set it explicitly if you want to display a different name for debugging purposes or when you create a higher-order component.

Reply:
Yeah I don't think it's needed for functional components.

Reply:
I read somewhere that it doesn't automatically grab the name if you are using arrow functions rather than named functions.  But then somewhere else said that Babel or other build tools take care of it.  So this was just sort of a "hey, in case things don't work later, park this is in the back of your brain".  I'm just going to mark this thread resolved.